packages/web/src/utils/HustlerConfig.ts

```diff
@@ -64,7 +64,7 @@ export const getRandomHustler = ({
   return {
     bgColor: bgColor || DEFAULT_BG_COLORS[getRandomNumber(0, DEFAULT_BG_COLORS.length - 1)],
     body: body || getRandomNumber(0, MAX_BODIES),
-    dopeId: dopeId || getRandomHustlerId(),
+    dopeId: dopeId || '42',
```

Comment:
@smakosh  why are we selecting "42" as a random hustler ID here? Doesn't make sense on first glance.

Reply:
That seems to fix the issue, when I use that function, it doesn't use the passed `dopeId`